File: 2974-MinimumNumberGame/2974-MinimumNumberGame.js
// Last updated: 3/17/2026, 10:48:51 AM
1/**
2 * @param {number[]} nums
3 * @return {number[]}
4 */
5var numberGame = function(nums) {
6    nums.sort((a,b)=>a-b);
7
8    for(let i=0;i<nums.length;i+=2){
9        let temp = nums[i];
10        nums[i] = nums[i+1];
11        nums[i+1] = temp;
12    }
13    return nums;
14};

File: 2974-MinimumNumberGame/2974-MinimumNumberGame.cpp
// Last updated: 3/17/2026, 10:50:40 AM
1class Solution {
2public:
3    vector<int> numberGame(vector<int>& nums) {
4        sort(nums.begin(),nums.end());
5
6        for(int i=0;i<nums.size();i+=2){
7            int temp = nums[i];
8            nums[i] = nums[i+1];
9            nums[i+1] = temp;
10        }
11
12        return nums;
13    }
14};